x-pack/qa/repository-old-versions-compatibility/src/javaRestTest/java/org/elasticsearch/oldrepos/Snapshot.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.oldrepos;
+
+/**
+ * This is a mapping to the created snapshot zip files, covering test cases for "old repositories"
+ */
+public enum Snapshot {
```

Comment:
```suggestion
/**
* Snapshot zip file names for archive and searchable snapshot tests
*/
public final class TestSnapshotCases {
  //Index created in vES_5 - Basic mapping
  public static final String ES_VERSION_5 = "5";
  // Index created in vES_5 - Custom-Analyzer - standard token filter
  public static final String ES_VERSION_5_STANDARD_TOKEN_FILTER = "5_custom_analyzer"
  ...
}
```

Let's make this a simple static class with some static string members. The descriptions are not anything more than what we could provide with a simple java comment.

Reply:
Thank you, @benwtrent, for reviewing this. I'll update the code following your approach

Reply:
Done!